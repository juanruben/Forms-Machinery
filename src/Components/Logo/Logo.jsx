import React from 'react';
import PropTypes from 'prop-types';
import image from '../../Assets/images/logo.png';

const Logo = (props) => {
    const { padding, maxWidth } = props;
    const divStyle = {
        padding,
        textAlign: 'center',
    };
    const imgStyle = {
        maxWidth,
    };
    return (
        <div style={divStyle}>
            <img style={imgStyle} src={image} alt="logo icafal" />
        </div>
    );
};

Logo.propTypes = {
    padding: PropTypes.number,
    maxWidth: PropTypes.number,
};

Logo.defaultProps = {
    padding: 0,
    maxWidth: 80,
};

export default Logo;
