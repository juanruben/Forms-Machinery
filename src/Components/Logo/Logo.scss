.logo{
	padding: 40px;
	img{
		max-width: 200px;
	}
}