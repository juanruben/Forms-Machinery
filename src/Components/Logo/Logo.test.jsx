import React from 'react';
import { shallow } from 'enzyme';
import Logo from './Logo';

describe('Logo', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Logo />);
    });

    it('should render correctly', () => expect(wrapper.debug()).toMatchSnapshot());

    it('should render div', () => {
        expect(wrapper.find('div')).toHaveLength(1);
    });

    it('should render img', () => {
        expect(wrapper.find('img')).toHaveLength(1);
    });
});
