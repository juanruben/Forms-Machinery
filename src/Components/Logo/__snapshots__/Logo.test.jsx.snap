// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render correctly 1`] = `
"<div style={{...}}>
  <img style={{...}} src=\\"logo.png\\" alt=\\"logo icafal\\" />
</div>"
`;
